src/lib/pages/frontsite/PatronProfile/PatronCurrentDocumentRequests/PatronCurrentDocumentRequests.js

```diff
@@ -228,15 +228,17 @@ class PatronCurrentDocumentRequests extends Component {
     this.setState({ isSuccessMessageVisible: false, successMessage: '' });
   };
 
-  onShowAll = () => {
+  onShowAll = (e) => {
+    e.preventDefault();
```

Comment:
fixes a warning in the console, `javascript:void(0)` should not be used in links, see below.